src/dialog-user/components/dialog-user/dialogUser.ts

```diff
@@ -94,13 +100,19 @@ export class DialogUserController extends DialogClass implements IDialogs {
    * @param value {any} This is the updated value based on the selection the user made on a particular dialog field
    */
   public updateDialogField(dialogFieldName, value) {
-    const refreshable = _.indexOf(this.refreshableFields, dialogFieldName);
     this.dialogFields[dialogFieldName].default_value = value;
     this.dialogValues[dialogFieldName] = value;
     this.saveDialogData();
-    if (refreshable > -1  && !this.areFieldsBeingRefreshed) {
+    if (this.fieldAssociations !== {}) {
```

Comment:
to make this code more readable, use lodash isEmpty  https://lodash.com/docs/4.17.4#isEmpty